tests/upgrade/upgrade_test.go

```diff
@@ -60,7 +77,8 @@ var _ = PDescribe("{UpgradeDowngradeVolumeDriver}", func() {
 		}
 
 		Step("start the upgrade of volume driver", func() {
-			err := Inst().V.UpgradeDriver(Inst().StorageDriverUpgradeVersion)
+			images := getImages(Inst().StorageDriverUpgradeVersion)
```

Comment:
Update documentation for `StorageDriverUpgradeVersion` on the expected CSV format

Reply:
@thiguetta let's handle the above comment and then we should be good to merge.